docs/index.md

```diff
@@ -1,97 +1,114 @@
-3. Get the Pull Request reviewed and merged.
 
 #### Build application
 
-1. Login to Jenkins and select "HMCTS - Labs" folder. Scan the organization by clicking on `Scan Organization Now`. New repository should be listed under repositories after scan finishes. Logs can be monitored under `Scan Organization Log`. Any GitHub repository that starts with `labs*` will be listed as part of this scan.
-
-   CFT: [jenkins](https://sandbox-build.platform.hmcts.net/job/HMCTS_LABS/)
+1. Login to Jenkins and select [HMCTS - Labs](https://sandbox-build.platform.hmcts.net/job/HMCTS_Sandbox_LABS/) folder.
+Scan the organization by clicking on `Scan Organization Now`.
+New repository should be listed under repositories after scan finishes.
+Logs can be monitored under `Scan Organization Log`.
+Any GitHub repository that starts with `labs*` will be listed as part of this scan.
 
 
 2. Run the jenkins pipeline against the `master` branch.
 
 #### Configure load balancing for HA
 
-1. We load balance across AKS clusters using `Azure Application Gateway`. Add a couple of lines of config for the application.
-
-   CFT:  [config file](https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/backend_lb_config.yaml)
+1. We load balance across AKS clusters using [Azure Application Gateway](https://docs.microsoft.com/en-us/azure/application-gateway/overview). Add a couple of lines of config for the application in [config file](https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/backend_lb_config.yaml).
 
+   ```yaml
+   #labs
+      - product: "labs"
+        component:     # GitHub repository name without "labs" prefix, e.g. `GithubUsername`
+   ```
      
-           #labs
-               - product: "labs"
-                 component:     #githubreponame without "labs" prefix
-   
 #### Deploy application
 
 1. We practise [GitOps](https://www.weave.works/technologies/gitops/) for application deployment to Kubernetes.
 
-   CFT: Read the documentation and follow the instructions [here]( https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md)
+   Follow the app deployment [guide](hmcts/cnp-flux-config@master/docs/app-deployment-v2.md#add-a-new-application) in cnp-flux-config.
 
 #### Access application
 
 1. Access the deployed application using the URL.
 
+   ```
+   http://<product>-<component>-sbox.service.core-compute-sandbox.internal 
+   ```  
    
-      http://<product>-<component>-sbox.service.core-compute-sandbox.internal   
-   
-#### Customize application
+#### Customise application
 
-1. Helm Charts can be customised by updating corresponding values file for each environment. Values files located under `/charts/<repo-name>/values.<ENV>.template.yaml`  
- 
-2. Environment variables can be passed by updating the corresponding values file in Helm chart. 
+1. We are going to customise the application by changing default landing page for the application by passing environment variables. 
```

Comment:
this is not a step, also in markdown i believe you can just use 1. for all numbers and it will automatically number them
```suggestion
We are going to customise the application by changing default landing page for the application by passing environment variables. 
```